### ISSUE TITLE
Train a new version of Deadcell with tuning

### ISSUE THREAD

**Issue by hiydavid** [hiydavid/numerai-models#106]
TODO:
- Use lightgbm trainer class (not sklearn API)
- Train 10000 rounds
- Implement grid search CV
- Implement early stopping
- Evaluate on validation set
- Neutralize top 10% features
- Submit for diagnosis
